### PR TITLE
automatic hex upload

### PR DESCRIPTION
**Pull request by nathanmonteleone**

### CODE REVIEW

File: .travis/upload.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+  #!/bin/bash
+
+set -e
+
+# Upload packages to hex
+# Only do so when on master
+
+if [[ ! -z "$TRAVIS_TAG" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+  mix hex.publish --organization simplifi --yes
```
Comment (LiruMouse) [inline .travis/upload.sh]: Not certain this sifi-specific bit is necessary, see [co_mix's `.travis/upload.sh`](https://github.com/simplifi/co_mix/blob/main/.travis/upload.sh)
```suggestion
  mix hex.publish --yes
```

File: .travis/upload.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+  #!/bin/bash
+
+set -e
+
+# Upload packages to hex
+# Only do so when on master
```
Comment (LiruMouse) [inline .travis/upload.sh]:
```suggestion
# Only do so when a new version is tagged
```

**Review by LiruMouse — APPROVED** [review]
:shipit:

**Review by joshuawscott — APPROVED** [review]
Should we bump the version here so that we can

**Comment by LiruMouse** [comment]
> Should we bump the version here so that we can

Versioning is tag-based now, just tag upon merge into `master`.

**Pull request merged**